crud/student-crud.js

```diff
@@ -0,0 +1,14 @@
+const { db, sequelize } = require(__dirname + "/../models/index.js");
+const Student = db["Student"];
```

Comment:
Instead of `hard-coding` the model name, we can create a json file that maps model name to file name in crud folder and import the model name from that json file depending on the name of our crud file

Reply:
`modelNames.json ` has been added and changes have been made in `student-crud.js` as per requirements.